apel/db/loader/loader.py

```diff
@@ -189,11 +189,34 @@ def load_msg(self, msg_text, signer):
         Load one message (i.e. one file; many records)
         from its text content into the database.
         '''
+        record_type_dict = {
```

Comment:
It's fairly obvious that this is a dictionary, so the name should just be something like `record_types`.